tests/kernel/arm_zero_latency_irqs/src/main.c

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
why is it split into two files?

Reply:
I suppose, I just followed the arm_irq_vector_table test structure